http/api/transport.go

```diff
@@ -6,6 +6,7 @@ package api
 import (
 	"context"
 	"encoding/json"
+
```

Comment:
here too

Reply:
probably fixed by rebasing

Reply:
The change was associated with a merge from master into the branch to keep it up to date.